### DISCUSSION TITLE
How can I change the currency format

### DISCUSSION THREAD

**Discussion by de-raaf-media** [duncanmcclean/simple-commerce#849] [Help] [answered]
Hi, maybe this is documented somewhere but I cannot seem to find this. Is it possible to change the currency notation? In The Netherlands cents a comma is used as a decimal separator and a period is used as a thousands separator.

So that would result in something like
€1.234,00

Regards,
Mark

**Comment by duncanmcclean** [comment, marked as answer]
Hey!

If you update to [v4.5.7](https://github.com/duncanmcclean/simple-commerce/releases/tag/v4.5.7), you can now customise these formatting symbols.

You can configure it in your Simple Commerce config file:

```php
// config/simple-commerce.php

'sites' => [
    'default' => [
        'currency' => 'GBP',

        'currency_formatting' => [
            'thousand_separator' => '.',
            'decimal_separator' => ',',
        ],
    ],
],
```

Hopefully that works for you!

**Reply by de-raaf-media** [reply]
Wow! Thanks for this great solution @duncanmcclean!
